dev/google-drive-files/generate-file-ids.rb

```diff
@@ -28,22 +33,27 @@ def list_files(drive, folder_id, parent_path: "", page_token: nil)
     list_files(
       dirve,
       folder_id,
+      version,
       parent_path: parent_path,
       page_token: response.next_page_token
     )
   end
 
   response.files.each do |item|
     if item.mime_type == "application/vnd.google-apps.folder"
-      list_files(drive, item.id, parent_path: "#{parent_path}#{item.name}/")
-    else
+      list_files(drive, item.id, version, parent_path: "#{parent_path}#{item.name}/")
+    elsif output?(item.name, version)
       puts "#{item.id}\t#{parent_path}#{item.name}"
     end
   end
 end
 
 # The JSON file for the service account is specified via the environment variable GOOGLE_APPLICATION_CREDENTIALS.
 # https://github.com/googleapis/google-auth-library-ruby?tab=readme-ov-file#example-service-account
+if ARGV.size != 1
+  puts "Usage: #{$0} NEXT_VERSION"
+  exit false
+end
 drive = Google::Apis::DriveV3::DriveService.new
 drive.authorization = Google::Auth::ServiceAccountCredentials.from_env(scope: SCOPES)
-list_files(drive, OPENARM_HARDWARE_FOLDER_ID)
+list_files(drive, OPENARM_HARDWARE_FOLDER_ID, $ARGV[0])
```

Comment:
Do we need `$` here?


```suggestion
list_files(drive, OPENARM_HARDWARE_FOLDER_ID, ARGV[0])
```